main.go

```diff
@@ -71,16 +70,13 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
 	opts := &ctrl.Options{
+		LeaderElection:     cl.EnableLeaderElection,
```

Comment:
what about configuration of the Leaseduration, RetryPeriod etc'? We are going to use the defaults?

Reply:
Upstream yes, we only started specifying those to match what OpenShift operators generally do.